optimistix/_solver/best_so_far.py

```diff
@@ -113,7 +113,16 @@ def postprocess(
         tags: frozenset[object],
         result: RESULTS,
     ) -> tuple[Y, Aux, dict[str, Any]]:
-        return state.best_y, state.best_aux, {}
+        f, aux = fn(y, args)
+        loss = self._to_loss(y, f)
+
+        best_f, best_aux = fn(state.best_y, args)
+        best_loss = self._to_loss(state.best_y, best_f)
```

Comment:
Agreed, I definitely don't understand these lines. What do you think is going on?